openedx/frontend-app-learner-dashboard
# Jkantor/notices

- Add notices to learner home.

Testing instructions:
 - set up https://github.com/edx/platform-plugin-notices for running locally
 - create a notice (I would suggest looking at the read replica and copying the 2u notice, the structure of the model data is a little confusing)
 - Navigate to the learner home. You should be redirected to the notice.
